api4/admin.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package api4
+
+import (
+	"net/http"
+
+	l4g "github.com/alecthomas/log4go"
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/mattermost/mattermost-server/utils"
+)
+
+func (api *API) InitAdmin() {
```

Comment:
I don't think we need a separate path for admin routes. I think it fits better under the 'users' path as `PUT /api/v4/users/{user_id}/auth`